themes/default/content/blog/kubernetes-v4-shared-resource-management-with-server-side-apply/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Kubernetes 4.0: Shared Resource Management With Server-Side Apply"
```

Comment:
```suggestion
title: "Kubernetes Native Provider 4.0: Build on Kubernetes Collaboratively and Safely with Shared Resource Management With Server-Side Apply"
```

Reply:
suggested edit - suggest bringing in the benefit of SSA into the title somehow since this is the main update for this version. feel free to edit a bit

Reply:
Added my own proposal in a new commit which fits in 60 characters.  @gthuang if you have additional suggested changes that fit in 60 characters, feel free to suggest them.